examples/dreambooth/train_dreambooth_lora_sana.py

```diff
@@ -979,10 +982,10 @@ def main(args):
         )
 
     # VAE should always be kept in fp32 for SANA (?)
-    vae.to(dtype=torch.float32)
+    vae.to(accelerator.device, dtype=torch.float32)
```

Comment:
This is not needed. As we conditionally put the VAE on and off the accelerator device.